tests/e2e/service_test.go

```diff
@@ -13,7 +13,7 @@ var _ = Describe("odoServiceE2e", func() {
 
 	Context("odo service creation", func() {
 		It("should be able to create a service", func() {
-			runCmd("odo service create mysql-persistent")
+			runCmd("odo service create mysql-persistent -w")
```

Comment:
Maybe one service without the wait flag?

Reply:
Good point, but we already have tests that don't use the flag, they just don't show up in the diff :wink:

Reply:
OK found it. Missed those tests in the first look up. Thanks :+1:

Reply:
Np :)